test/streaming/get/test_network_consumer.py

```diff
@@ -581,11 +609,21 @@ def responses(request: GetIdentifiedObjectsRequest) -> Generator[GetIdentifiedOb
     return responses
 
 
-def _create_container_responses(ns: NetworkService, mrids: Optional[Iterable[str]] = None) \
+def _create_container_responses(
+    ns: NetworkService,
+    mrids: Optional[Iterable[str]] = None,
+    expected_include_energizing_containers: Optional[int] = None,
+    expected_include_energized_containers: Optional[int] = None
+) \
     -> Callable[[GetEquipmentForContainersRequest], Generator[GetEquipmentForContainersResponse, None, None]]:
     valid: Dict[str, EquipmentContainer] = {mrid: ns[mrid] for mrid in mrids} if mrids else ns
 
     def responses(request: GetEquipmentForContainersRequest) -> Generator[GetEquipmentForContainersResponse, None, None]:
+        if expected_include_energizing_containers is not None:
+            assert request.includeEnergizingContainers == expected_include_energizing_containers
+        if expected_include_energized_containers is not None:
+            assert request.includeEnergizedContainers == expected_include_energized_containers
```

Comment:
There may be a better place to put these assertions, and I would have them on the channel rather than the server if possible. Ultimately, I just want to ensure that the linked container params are sent across in my new test.